app/models/branch_of_service_contact.rb

```diff
@@ -0,0 +1,14 @@
+class BranchOfServiceContact < ApplicationRecord
+  validates :branch, presence: true
+
+  def retiree_contact_info?
+    retiree_dsn || 
+    retiree_tel_comm || 
+    retiree_tel_tollfree || 
+    retiree_email || 
+    retiree_fax_dsn || 
+    retiree_fax_comm || 
+    retiree_fax_tollfree || 
+    retiree_post
+  end
+end
```

Comment:
☝️ Also Also: With the addition of these methods in the model, you can make use of that stubbed `spec/models/branch_of_service_contact_spec.rb` file.